Updated PhysX Wind Provider topic. Updated steps moved/replaced images. (#1473)
o3de/o3de.org: content/docs/learning-guide/tutorials/physx/wind-provider.md

    {{< image-width src="/images/learning-guide/tutorials/physx/wind-collider-setup.png" width="900" alt="Setting up a PhysX collider for a force region" >}}

## Create a PhysX Force Region

In this section, you'll set up a **PhysX Force Region** that generates a wind force.

1. Add a PhysX Force Region component to the entity. This component creates the wind force. To configure the component, do the following:

1. Choose the {{< icon "add.svg" >}} **Add** button to add a new force.

1. In the **Direction** property, set the **Y** component a value of `-1.0` and set the **Z** component to `0.0` to create a direction for the wind force.

1. In the **Magnitude** property, set a value of `10.0` to create magnitude for the wind force.

    The PhysX collider box displays cones representing the wind force direction.

    {{< image-width src="/images/learning-guide/tutorials/physx/wind-force-region-setup.png" width="900" alt="Setting up a PhysX collider for a force region." >}}

## Add a Cloth prefab

> 1. To test the wind provider, add prefab with an NVIDIA Cloth mesh. In **Asset Browser**, navigate to `Gems\NvCloth\Assets\prefabs\Cloth`, locate the `cloth_locked_edge.prefab`, and then **drag** it into the viewport.
> 
> 1. Use the **Move** tool to place the cloth prefab. If you are using the **Local wind tag** property, you must place the cloth asset inside the PhysX Collider volume of the wind provider entity.

Reviewer: ```suggestion
1. To test the wind provider, add a prefab with an NVIDIA Cloth mesh. In **Asset Browser**, navigate to `Gems\NvCloth\Assets\prefabs\Cloth`, locate `cloth_locked_edge.prefab`, and then drag it into the viewport.

1. Use the Move tool to place the cloth prefab. If you are using the **Local wind tag** property, you must place the cloth asset inside the PhysX Collider volume of the wind provider entity.
```
* Line 67:
  * Add missing word "a" ("add a prefab")
  * Cut "the" before the prefab file name. ("locate [file name]") 
  * Unless "drag" is a UI text label, drop the **bold** formatting.
* Line 69
  * Second occurrence of "Move tool". You can drop the bold formatting for this occurrence.
  * For consistency with the section header ("Add a Cloth prefab"), use either "Cloth prefab" or "cloth prefab" for both occurrences. Your call.

        
    1. In Entity Outliner, double click the {{< icon "prefab.svg" >}} **cloth_locked_edge** prefab to edit it in Focus Mode.

    2. Click the {{< icon "entity.svg" >}} **cloth_locked_edge** child entity to select it.

        ![Open the cloth prefab for editing in Focus Mode.](/images/learning-guide/tutorials/physx/edit-cloth-locked-edge-prefab.png)

    3. With the {{< icon "entity.svg" >}} **cloth_locked_edge** entity selected, in Entity Inspector, in the Cloth component, expand the **Wind** property group and disable the **Enable local wind velocity** toggle.

        {{< image-width src="/images/learning-guide/tutorials/physx/disable-local-wind-velocity.png" width="450" alt="Setting up a PhysX collider for a force region" >}}

## Test the wind simulation

Finally, you can test the simulation. With the **cloth_locked_edge** prefab open for editing in Focus Mode, do the following:

> 1. At the top of the Cloth component, enable the **Simulate in editor** toggle. As the simulation begins, the cloth object might flip and stretch wildly, but it quickly settles into a breezy wind simulation.

Reviewer: ```suggestion
1. At the top of the Cloth component, turn on **Simulate in editor**. As the simulation begins, the cloth object might flip and stretch wildly, but it quickly settles into a breezy wind simulation.
```
For consistency with the previous edit of changing "disable" to "turn off", consider using "turn on" here (and dropping "toggle").

> 1. With the toggle enabled, the wind simulation plays while in editor mode and you can adjust the simulation by modifying the various properties of the Cloth component. Try adjusting the following properties in the Cloth component:

Reviewer: ```suggestion
1. The wind simulation plays while in editor mode. You can adjust the simulation by modifying the various properties of the Cloth component. Try adjusting the following properties in the Cloth component:
```
Trims for brevity, sentence break for flow.

    * **Air drag coefficeint**
    * **Air lift coefficeint**
    * **Air Density**

>     {{< image-width src="/images/learning-guide/tutorials/physx/cloth-simulate-in-editor.png" width="450" alt="Simulating Cloth in editor." >}}

Reviewer: ```suggestion
    {{< image-width src="/images/learning-guide/tutorials/physx/cloth-simulate-in-editor.png" width="450" alt="Simulating cloth in editor." >}}
```

> The following video shows the results of the above Cloth component settings.

Reviewer: ```suggestion
The following video shows the results of using the Cloth component settings shown in the preceding image.
```
Edits for accessibility considerations:
* Expanded the description slightly to clarify that the settings referenced appear in the image (not written out).
* Dropped "above", since a reader using a screen reader might not make the same spatial distinction.

    * Choose the {{< icon "entity.svg" >}} **cloth_locked_edge** child entity to select it.

        ![Open the cloth prefab for editing in Focus Mode.](/images/learning-guide/tutorials/physx/edit-cloth-locked-edge-prefab.png)

    * With the {{< icon "entity.svg" >}} **cloth_locked_edge** entity selected, in Entity Inspector, in the Cloth component, expand the **Wind** property group and turn off **Enable local wind velocity**.

        {{< image-width src="/images/learning-guide/tutorials/physx/disable-local-wind-velocity.png" width="450" alt="Turning off local wind velocity in a Cloth component." >}}

## Test the wind simulation

Finally, you can test the simulation. With the **cloth_locked_edge** prefab open for editing in Focus Mode, do the following:

1. At the top of the Cloth component, turn on **Simulate in editor**. As the simulation begins, the cloth object might flip and stretch wildly, but it quickly settles into a breezy wind simulation.

1. The wind simulation plays while in editor mode. You can adjust the simulation by modifying the various properties of the Cloth component. Try adjusting the following properties in the Cloth component:

>     * **Air drag coefficeint**
>     * **Air lift coefficeint**

Reviewer 2: ```suggestion
    * **Air drag coefficient**
    * **Air lift coefficient**
```
